### DISCUSSION TITLE
文档中的阿里网盘刷新令牌扫码后无回应

### DISCUSSION THREAD

**Discussion by pickarm** [AlistGo/alist#784] [Q&A] [answered]
### Alist Version / Alist 版本

v2.2.0

### Describe the bug / 问题描述

刷新令牌扫码登陆后无反应，尝试更换流量，WiFi，挂梯后均无法解决

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by xhofe** [comment, marked as answer]
扫码之后点了吗
另外这种问题请去discussion，不要提issue。

**Reply by pickarm** [reply]
好的，不好意思，第一次使用issue，下次会注意的，点了按钮并无反应，电脑从chrome更换为edge和使用手机夸克问题依旧。

**Reply by xhofe** [reply]
那你只能查找日志了 我这边使用无任何问题。

**Reply by pickarm** [reply]
好的，谢谢
